kuka_iiqka_eac_driver/package.xml

```diff
@@ -18,6 +18,7 @@
   <depend>kuka_drivers_core</depend>
   <depend>hardware_interface</depend>
   <depend>pluginlib</depend>
+  <depend>kuka-external-control-sdk</depend>
```

Comment:
I think we can only add this if the kuka-external-control-sdk is a ros package or has a rosdep key

Reply:
Yes, CI fails because of this

```
  ERROR: the following packages/stacks could not have their rosdep keys resolved
  to system dependencies:
  kuka_iiqka_eac_driver: Cannot locate rosdep definition for [kuka-external-control-sdk]
```